support/ci/install-nightly-rustfmt.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script can be run standalone, but it can also be sourced from rustfmt.sh.
+# If it's sourced, then $rustup is already defined for us. If it's not, we need
+# to make sure we have it.
+if [ -z "${rustup:-}" ]; then
+  rustup=$(command -v "rustup")
```

Comment:
There's no need for the subsequent check as your script will exit due to `set -e` if `$(command -v "rustup")` fails.

Reply:
Ah, good point.